<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_taxi_fare</class>
 <widget class="QDialog" name="change_taxi_fare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Fare</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton{
    
	color: rgb(74, 3, 255);
    font:black;
}
QRadioButton::hover{
   color:black;
}
QRadioButton::pressed{
   color:black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Increase fare per km by -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_inc_2">
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string> 2 Rs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_inc_3">
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>3 Rs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_inc_5">
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string> 5 Rs.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Decrease fare per km by -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_dec_2">
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>2 Rs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_dec_3">
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>3 Rs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_dec_5">
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>5 Rs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: #1E90FF;
  color: white;
  border-radius: 5px;
  padding: 3px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color: rgb(255, 232, 115);
  font-style: italic;
  border: 2px solid white;
  color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
font-color: black;
  color: white;
  border: 2px solid white;
}
</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">QPushButton {
  background-color: #1E90FF;
  color: white;
  border-radius: 5px;
  padding: 3px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color:rgb(166, 98, 200);
  font-style: italic;
  border: 2px solid white;
  color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
font-color: black;
  color: white;
  border: 2px solid white;
}
</string>
     </property>
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
